### PR TITLE
Remove r4407 changelogs, adjust changelog timeout

### PR DESCRIPTION
**Pull request by celotajstg**
## About The Pull Request

This PR ~~adds changelogs from 2008 from ([source](https://wiki.ss13.co/Changelog#April_2008))~~ removes changelogs from r4407 and earlier, as well as adjusts the changelog fetch request re-attempt timeout to be more sensible.

## Why It's Good For The Game

Faster changelog loading, and we finally stop crediting GoonStation 13 changes to ourselves.

## Changelog
:cl: Celotajs
qol: The changelog re-request delay has been heavily reduced
qol: Changelogs from r4407 (April 2010) and earlier have been removed, and a link to the GoonStation 13 wiki page for the old changelogs has been added to the footer in the changelog popup. This means we finally won't claim GoonStation13's changes as our own.
/:cl:

### CODE REVIEW

**Comment by maxymax13** [comment]
logs from before july 2010 should be labeled [Goon Station 13](https://github.com/goonstation/goonstation-r4407/blob/master/icons/changelog.html#L4)

**Comment by maxymax13** [comment]
you kinda reached all the way into 2011 there buddy

File: html/changelogs/archive/2010-04.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@
       \ quartermasters to automatically ship crates to different parts of the station.\n\
       \t  </li>\n\t</ul>\n  "
 2010-04-25:
-  /tg/station ss13 team:
+  GoonStation 13 Development Team:
```
Comment (maxymax13) [inline html/changelogs/archive/2010-04.yml:30]: this is where it ends

**Comment by maxymax13** [comment]
okay well now everything's corrected except now the very last r4407 change is now labelled /tg/station 13 team again noooooooooo

**Comment by celotajstg** [comment]
> okay well now everything's corrected except now the very last r4407 change is now labelled /tg/station 13 team again noooooooooo

Misunderstood you, will change it

**Comment by maxymax13** [comment]
is all good now 👌

**Comment by optimumtact** [comment]
You're touching a very sensitive topic here

**Comment by cacogen** [comment]
I always thought /tg/station branched off GoonStation in late 2010, because that's where the Google Code SVN started. The first SVN commit was on [24 Aug](https://github.com/tgstation/tgstation/commit/757ea7b077f6b86ab6e4392808e773eea975338d). But this only goes to 25 April. The changelogs continue from then past 24 Aug. Do you have some way of knowing they forked earlier?

**Comment by ZeWaka** [comment]
> Do you have some way of knowing they forked earlier?

Forked? From what, ourselves?

**Comment by maxymax13** [comment]
> 
> 
> I always thought /tg/station branched off GoonStation in late 2010, because that's when the Google Code SVN started. The first SVN commit was on [24 Aug](https://github.com/tgstation/tgstation/commit/757ea7b077f6b86ab6e4392808e773eea975338d). But this only goes to 25 April. The changelogs on the page you linked continue from then past 24 Aug. Do you have some way of knowing they forked earlier?

the version of the code is from april, the first addition to the changelog diverging from the goon one is from july

**Comment by cacogen** [comment]
Oh, I see. The existing changelogs go back further than the beginning of /tg/station. Years and years ago I read them but I'd forgotten that. Thanks for explaining the methodology.

**Comment by Iamgoofball** [comment]
Have we audited this for slurs and stuff?

Old-old goon was rather uhh, gamer :tm:.

**Comment by maxymax13** [comment]
image
image
looks like that was already taken care of

**Comment by celotajstg** [comment]
> You're touching a very sensitive topic here

Should I show those commits as /tg/station 13 ss13 team still?

**Comment by cacogen** [comment]
It's unclear why it's a sensitive topic and to whom but it would be more insensitive to attribute those changes to the wrong people

**Comment by optimumtact** [comment]
Okay we've discussed it, please remove all changelogs from pre r4407, i.e changes not made by tgstation coders.

Then update the changelog text to mention goonstation, thank them for the r4407 release and link to their wiki changelog.

**Comment by maxymax13** [comment]
**Living under a floor tile since 2546? Click here!**

**Comment by celotajstg** [comment]
Done

**Comment by optimumtact** [comment]
thanks for being patient

**Pull request merged**